frontend/src/app/Settings/Users/ManageUsers.tsx

```diff
@@ -37,7 +37,7 @@ interface Props {
   updateUserRole: (variables: any) => Promise<any>;
   addUserToOrg: (variables: any) => Promise<any>;
   deleteUser: (variables: any) => Promise<any>;
-  getUsers: () => void;
+  getUsers: () => Promise<any>;
 }
 
 export type SettingsUsers = { [id: string]: SettingsUser };
```

Comment:
Wondering whether the backend should be sorting them for us here? Maybe we don't get to a point where users are too big and need server-side pagination.